applications/LinearSolversApplication/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ message( "**** configuring KratosLinearSolversApplication ****" )
 ################### PYBIND11
 include(pybind11Tools)
 
-add_definitions( -DEIGEN_DEFAULT_TO_ROW_MAJOR -DEIGEN_MPL2_ONLY )
+# TODO add_definitions( -DEIGEN_MPL2_ONLY)

```

Comment:
* Spectra internally uses the Eigen::Cholesky, that however is not available with MPL2_ONLY. I will try to write an interface to another Eigen solver instead, ev, even MKL
* We have tried Spectra before without success: I think it was because it does not work with `-DEIGEN_DEFAULT_TO_ROW_MAJOR`. That does not have to be used however, as long as we specify the RowMajor ordering explicitly when interfacing Eigen matrices with Kratos matrices.